lib/github.sh

```diff
@@ -214,8 +215,10 @@ strap::github::api::token::create() {
     if command -v gio >/dev/null; then
       gio open https://github.com/settings/tokens
     # Mac
-    else
+    elif command -v open >/dev/null; then
       open https://github.com/settings/tokens
+    else
+      echo "Cannot detect command to open browser! Please open https://github.com/settings/tokens"
     fi
     echo "One or more of your Github Org requires you to enable the token before using. See more details on https://help.github.com/en/articles/authorizing-a-personal-access-token-for-use-with-a-saml-single-sign-on-organization."
```

Comment:
nit: `One or more of your Github Org requires you to enable the token before using. See more details on` to `One or more of the Github Orgs you belong to requires that you to enable an SSO personal access token before proceeding. See more details here: `